### PR TITLE
update(buttons): add split / drop-down buttons

### PR DESCRIPTION
**Pull request by technabors**
Add split button and dropdown buttons to `buttons.md`. Updates as below

- Remove `{:width="100%"}` distinctions to prevent poor scaling on existing images
- Added H2 variations, with H3 for Drop-down button, H4 for specs
- Added H3 for split drop-down button, H4 for specs
- New images for both sections provided in SVG format.
- Final image exported as both SVG and `@2x` png format for comprisons

### CODE REVIEW

File: _component_design/buttons.md [hunk 1]
```diff
@@ -191,13 +191,87 @@ When submitting forms:
...
+</div>
+
+## Variations
+{: .hxSubSectionTitle}
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Drop-down buttons
+
+A drop-down button is preferred when multiple actions are available, with no designated primary action. All actions are  accessed through the drop-down menu.
```
Comment (robb-romans) [inline _component_design/buttons.md]: rm extra space "are accessed"

File: _component_design/buttons.md [hunk 2]
```diff
@@ -191,13 +191,87 @@ When submitting forms:
...
+{% endfigure %}
+{% endcolumn %}
+</div>
+
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Split drop-down buttons
+
+Split dropdown buttons are preferred when more than one action is associated with a button. The primary action is visible and available on the button itself, while the secondary actions are accessed through the drop-down menu.
```
Comment (robb-romans) [inline _component_design/buttons.md]: Split drop-down buttons...

Comment (araiford) [inline _component_design/buttons.md]: Consider adding a paragraph under this referencing the Actions Menu.

Something like:
See the [Actions Menu](url) for drop-down design specifications.

File: _component_design/buttons.md (hunk 1 above)
Comment (technabors) [reply _component_design/buttons.md]: updated.

File: _component_design/buttons.md (hunk 2 above)
Comment (technabors) [reply _component_design/buttons.md]: updated.

Comment (technabors) [reply _component_design/buttons.md]: added in lower case form.

File: _component_design/buttons.md [hunk 3]
```diff
@@ -191,13 +191,87 @@ When submitting forms:
...
 {% endfigure %}
 {% figure [caption:"Primary and tertiary button set"] [class:"image bg-light border"] %}
-![]({{site.url}}/assets/images/components/inputs-and-controls/buttons/buttons-sets-3.svg){:width="100%"}
+![]({{site.url}}/assets/images/components/inputs-and-controls/buttons/buttons-sets-3.svg)
+{% endfigure %}
+{% endcolumn %}
+</div>
+
+## Variations
+{: .hxSubSectionTitle}
```
[the remark this answers is not in the document]
Comment (technabors) [reply _component_design/buttons.md]: removed, can tell this is an older file =p

**Pull request merged**